examples/mplot3d/custom_shaded_3d_surface.py

```diff
@@ -13,8 +13,8 @@
 import numpy as np
 
 # Load and format data
-filename = cbook.get_sample_data('jacksboro_fault_dem.npz', asfileobj=False)
-with np.load(filename) as dem:
+with cbook.get_sample_data('jacksboro_fault_dem.npz') as file, \
+     np.load(file) as dem:
```

Comment:
I find this hard to read, but not impossible.  Sure you don't want to just double-with it?

Reply:
@tacaswell approves so I guess its good...